src/seedu/addressbook/data/menu/Price.java

```diff
@@ -40,6 +40,11 @@ public double convertValueOfPricetoDouble(){
     public static String convertPricetoString(double priceIndouble){
         String valueAsString = Double.toString(priceIndouble);
         String valueAsPrice = "$" + valueAsString;
+        //ensuring the final answer is always returned in 2 decimal places
+        int decimalIndex = valueAsPrice.indexOf(".");
+        if((valueAsPrice.substring(decimalIndex)).length()<3){
```

Comment:
Can you add another condition: if length>3 then only take the first 3 characters of the string (if the number of digits after the dot is greater than two, just in case we implement discount rate in the future that might give a divided number)